frontend/src/metabase/query_builder/components/expressions/Layout/Layout.tsx

```diff
@@ -0,0 +1,33 @@
+import cx from "classnames";
+import type { HTMLAttributes } from "react";
+
+import { Box, type BoxProps } from "metabase/ui";
+
+import S from "./Layout.module.css";
+
+type Props = BoxProps & HTMLAttributes<HTMLDivElement>;
+
+export function Layout(props: Props) {
+  return <Box {...props} className={cx(S.layout, props.className)} />;
+}
+
+function LayoutHeader(props: Props) {
+  return <Box {...props} className={cx(S.header, props.className)} />;
+}
+
+function LayoutMain(props: Props) {
+  return <Box {...props} className={cx(S.main, props.className)} />;
+}
+
+function LayoutSidebar(props: Props) {
+  return <Box {...props} className={cx(S.sidebar, props.className)} />;
+}
+
+function LayoutFooter(props: Props) {
+  return <Box {...props} className={cx(S.footer, props.className)} />;
+}
+
+Layout.Header = LayoutHeader;
```

Comment:
I think we don't normally do that and use components directly. This will make tree-shaking impossible